language/ir-testsuite/tests/commands/abort_in_module.mvir

```diff
@@ -0,0 +1,19 @@
+module M {
+    public foo() {
+        abort 22;
+    }
+
+}
+
+//! new-transaction
+
+import {{default}}.M;
+
+main() {
+    M.foo();
+    return;
+}
+
+// check: ABORTED
+// check: 22
+// check: "0xa4a46d1b1421502568a4a6ac326d7250::M::foo at offset 2"
```

Comment:
@vgao1996 is this safe? in other words is the default address used stable?
is there a way to use a well defined address?

Reply:
It is only safe in the sense that the random seed used to generate this address is fixed. Changes in the RNG implementation or generating another address before this will break it.

Currently aliases like `{{alice}}` cannot be used in checker directives so I'd recommend writing `// check: "::M::foo at offset 2"` instead.

Reply:
Another option would be setting the sender to `association` -- that should give us more stability since the address will then be 0xA550C18.